lib/base.js

```diff
@@ -62,12 +136,30 @@ Base.prototype.getId = function() {
     return this._id;
 };
 
+Base.prototype.getUser = function() {
+    return this._user;
+};
+
+Base.prototype.getUsername = function() {
+    if (this._user === null) {
+        return null;
+    }
+    return this._user.user.username;
```

Comment:
Hmm... I'm not sure I like the field being called `user` within `_user`. Maybe `this._user.fields` would make sense?